index.js

```diff
@@ -79,7 +79,8 @@ var WebDriverInstance = function (baseBrowserDecorator, args, logger) {
     log.debug('WebDriver config: ' + JSON.stringify(config));
     log.debug('Browser capabilities: ' + JSON.stringify(spec));
 
-    self.browser = wd.remote(config, 'promiseChain').init(spec);
+    var browser = wd.remote(config, 'promiseChain');
+    self.browser = browser.init(spec);
```

Comment:
Perhaps one of these could have a different name? The self.browser is not used anywhere other than this script as far as I'm aware, so having them both named browser can be confusing. I'd propose:

```
self.driver = wd.remote(config, 'promiseChain');
self.browser = self.driver.init(spec);
```